src/renderer.ts

```diff
@@ -196,10 +196,11 @@ const plainTextElement = (node: Node, key: number): ReactNode => {
 	}
 };
 
-const textElement = (format: Format, supportsDarkMode = true) => (
-	node: Node,
-	key: number,
-): ReactNode => {
+const textElement = (
+	format: Format,
+	supportsDarkMode = true,
+	isEditions = false,
```

Comment:
Given that `supportsDarkMode` was introduced to handle the fact that Editions doesn't, should we combine both these flags into the `isEditions` flag and just pass that to `Anchor`?

Reply:
Given that `supportsDarkMode` is used everywhere, I've changed the blockquote renderer code just to use this rather than replace it throughout with `isEditions`

Reply:
Wait, so we use `supportsDarkMode` to decide whether to show quote marks in light mode?

Reply:
When you put it like that it sounds kind of silly :) Fixed now so it's `isEditions` throughout.

Reply:
Nice. You know, given the migration I think it might be that we end up with a `Platform` type to handle these minor UI changes on different platforms. Something like:

```ts
enum Platform {
  Apps,
  Editions,
  Dotcom,
  Amp,
}
```

Then you can do stuff like:

```ts
const styles = (platform: Platform): SerializedStyles => {
  switch (platform) {
    case Platform.Apps:
      return someDarkModeStyles;
    case Platform.Editions:
      return someSpecialEditionsStyles;
    default:
      return someWebStyles;
  }
}
```

In a given component you could also swap in entirely different sub-components depending on the platform you're currently rendering.